backend/attendance-service/src/main/java/org/egov/web/models/AttendanceStaffSearchCriteria.java

```diff
@@ -3,17 +3,19 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.*;
 
+import java.util.List;
+
 @Getter
 @Setter
 @AllArgsConstructor
 @NoArgsConstructor
 @Builder
 public class AttendanceStaffSearchCriteria {
```

Comment:
Where is this class used? We do not have a Staff search endpoint.

Reply:
We are using staff search criteria to retrieve staff for the following checks:
1. Check if the staff is already enrolled to the register during staff enrollment
2. Check if the staff is already de-enrolled from the register during deenrollment and it is also used to get the existing staff row to update the de-enrollment date

Reply:
Still, don't keep this class in the web/models package. That package is reserved for the classes used in the API.

Reply:
new models package created for searchCriteria classes as per discussion